src/UserSettings.hs

```diff
@@ -18,7 +16,7 @@ import {-# SOURCE #-} Settings.Default
 
 -- | All build results are put into the 'buildRoot' directory.
 userBuildRoot :: BuildRoot
-userBuildRoot = BuildRoot "_build"
+userBuildRoot = error "build root not set" -- BuildRoot "_build"
```

Comment:
I think it is better to use the default value here, since we should pass in `--build-root` explicitly if we want to change it.

Reply:
Oh yes, that's just a leftover from some debugging I did. Nice catch :-)

Reply:
Looks like CI bots are broken by this `error`.

I hope this PR will actually fix the recent series of breakages due to `ghc-cabal`, so ideally we should be getting green light from CI :)